news/4.36/platform_isv.md

```diff
@@ -5,7 +5,25 @@
 ## Platform Changes
 -->
 
-<!--
 ---
 ## SWT Changes
--->
+
+### Lazy Handle Generation for Images and Fonts (Windows only)
+
+When creating resources like image`Image` and `Font`, the necessary OS handles used to be created eager upon instantiation of the resource.
```

Comment:
missing space before `Image`

Reply:
Thank you! Fixed with latest commit.
I also forgot to amend my proofreading changes, so the latest commit also fixes some typos.